src/sanitize.js

```diff
@@ -8,7 +8,10 @@ const clean = dirty => sanitizeHtml(dirty, {
 export const sanitizeElementName = clean;
 export const sanitizeAttrVal = clean;
 export const sanitizeAttr = clean;
-export const sanitizeJSON = dirty => JSON.stringify(dirty).replace(/</g, '\\u003c');
+export const sanitizeJSON = (dirty) => {
+  const result = JSON.stringify(dirty);
```

Comment:
Do you mean that the regex is slow? We are trying to avoid html injection with it.